lib/libtlspool.c

```diff
@@ -214,7 +173,7 @@ static pthread_mutex_t registry_lock = PTHREAD_MUTEX_INITIALIZER;
  * The return value is 0 on success, or -1 on failure; the most probable
  * cause for failure is
  */
-static int registry_update (int *reqid, struct registry_entry *entry) {
+int registry_update (int *reqid, struct registry_entry *entry) {
 	static int simu = -1;
```

Comment:
whoa, die code wil ik graag afschermen van gebruikers, daarom static.  Dit is in libtlspool, waar zichtbaarheid key is aan de integrale werking